### PR TITLE
Updated the client to support version 1.6

### PR DESCRIPTION
**Pull request by SirPoot-**
API Version 1.6 was recently released 

**Key changes**
- User account must be an 'API User' type
- Must include integration code with proper version reference in the soap header
- Cannot no longer use "auto-generated" WSDL file

In this request, I allowed you to enter the integration code when instantiating the client or you can set it after the client is instantiated and the version is dynamically determined based on the wsdl path. I've tested the following using API version 1.5 and [1.6.]

I've attached the latest API documentation for reference.
[T_WebServicesAPIv1_6.pdf](https://github.com/opendns/autotask-php/files/2789940/T_WebServicesAPIv1_6.pdf)

Let me know what you think or how you would like to move forward.

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -310,6 +319,10 @@ public function getInvoiceMarkup($invoiceId, $type)
 
     public function __doRequest($request, $location, $action, $version, $one_way = 0)
     {
+        if (strpos($action, 'getZoneInfo') === false && $this->version >= 1.6 && empty($this->__default_headers[0]->data['IntegrationCode'])) {
```
Comment (reynolek) [inline src/Client.php]: Formatting this line for readability.
```suggestion
        if (strpos($action, 'getZoneInfo') === false &&
            $this->version >= 1.6 &&
            empty($this->__default_headers[0]->data['IntegrationCode'])
        ) {
```

**Comment by reynolek** [comment]
Do we need to add more AutotaskObjects or is everything still represented?

**Comment by SirPoot-** [comment]
Totally agree with the version releases and we can address at that time. This is the first minor is years and do not see that many minors coming. They are in the process of developing an REST API to be released sometime at the end of this year. With that - they said are going to keep the SOAP available (pretty much forever) for the existing integrations. 

Yes - There are four new entities and a handful of existing entities have new fields available. I will submit another PR with those. Wanted to get this approved before working on that.

**Comment by SirPoot-** [comment]
I added the entities and updated fields bringing the API to version 1.6.1.

**Comment by reynolek** [comment]
Excellent -- thanks for adding the comments in the code to reference 1.6 being required for the new objects.

**Comment by reynolek** [comment]
Released as v0.3.0

**Pull request merged**